lib/src/components/NxTransferList/__tests__/TransferListHalf.test.tsx

```diff
@@ -273,7 +277,87 @@ describe('TransferListHalf', function() {
 
       component.find('.nx-transfer-list__move-all').simulate('click');
 
-      expect(onMoveAll).toHaveBeenCalledWith(new Set([1, 2, 4, 5]));
+      expect(onMoveAll).toHaveBeenCalledWith([1, 2, 4, 5]);
+    });
+  });
+
+  describe('reordering', function() {
+    it('renders move up and down buttons inside .nx-transfer-list__item in the correct state', function() {
+      const items = [{
+            id: 1,
+            displayName: 'top'
+          }, {
+            id: 3,
+            displayName: 'middle'
+          }, {
+            id: 4,
+            displayName: 'bottom'
+          }],
+          component = getMounted({ allowReordering: true, items }),
+          list = component.find('.nx-transfer-list__item-list');
+
+      const firstItem = list.find('.nx-transfer-list__item').at(0),
+          secondItem = list.find('.nx-transfer-list__item').at(1),
+          thirdItem = list.find('.nx-transfer-list__item').at(2),
+          firstItemMoveUpButton = firstItem.find('.nx-transfer-list__button--move-up').at(0),
+          firstItemMoveDownButton = firstItem.find('.nx-transfer-list__button--move-down').at(0),
+          secondItemMoveUpButton = secondItem.find('.nx-transfer-list__button--move-up').at(0),
+          secondItemMoveDownButton = secondItem.find('.nx-transfer-list__button--move-down').at(0),
+          thirdItemMoveUpButton = thirdItem.find('.nx-transfer-list__button--move-up').at(0),
+          thirdItemMoveDownButton = thirdItem.find('.nx-transfer-list__button--move-down').at(0);
+
+      expect(firstItem).toExist();
+      expect(firstItemMoveUpButton).toExist();
+      expect(firstItemMoveDownButton).toExist();
+      expect(firstItemMoveUpButton.hasClass('disabled')).toBeTruthy();
+      expect(firstItemMoveDownButton.hasClass('disabled')).toBeFalsy();
+
+      expect(secondItem).toExist();
+      expect(secondItemMoveUpButton).toExist();
+      expect(secondItemMoveDownButton).toExist();
+      expect(secondItemMoveUpButton.hasClass('disabled')).toBeFalsy();
+      expect(secondItemMoveDownButton.hasClass('disabled')).toBeFalsy();
+
+      expect(thirdItem).toExist();
+      expect(thirdItemMoveUpButton).toExist();
+      expect(thirdItemMoveDownButton).toExist();
+      expect(thirdItemMoveUpButton.hasClass('disabled')).toBeFalsy();
+      expect(thirdItemMoveDownButton.hasClass('disabled')).toBeTruthy();
     });
```

Comment:
Test that they have the right icons too

Reply:
> Test that they have the right icons too

Added tests for those in this commit: 774e9702b6a1123bac0705354974edd6087775a5

Reply:
> Added tests for those in this commit: 774e970

That test's name seems like it's copied from somewhere else

Reply:
> > Added tests for those in this commit: [774e970](https://github.com/sonatype/sonatype-react-shared-components/commit/774e9702b6a1123bac0705354974edd6087775a5)
> 
> That test's name seems like it's copied from somewhere else

Yup! That was fixed here: 8da7e29ba73263e6ce9f96e12d36ba9915305d0f